### PR TITLE
fc: refresh page

### PR DESCRIPTION
**Pull request by Managor**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by spageektti** [comment]
Hi @Managor . What do you think about adding 
```fc --help```
example to the end?

**Comment by Managor** [comment]
Sure, why not.

File: pages/common/fc.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # fc
 
-> Open the most recent command and edit it.
+> Open the most recent command for edit and then run it.
```
Comment (gutjuri) [inline pages/common/fc.md:3]:
```suggestion
> Open the most recent command for editing and then run it.
```

**Review by gutjuri — APPROVED** [review]
Thanks, one suggestion

**Review by spageektti — APPROVED** [review]

**Pull request merged**